pom.xml

```diff
@@ -1694,5 +1727,39 @@
         <module>deploy/kubernetes</module>
       </modules>
     </profile>
+    <profile>
+      <id>hadoop2.8</id>
+      <properties>
+        <hadoop.version>2.8.5</hadoop.version>
+        <hadoop.short.version>2.8</hadoop.short.version>
+      </properties>
+    </profile>
+    <profile>
+      <id>hadoop3.2</id>
+      <properties>
+        <hadoop.version>3.2.1</hadoop.version>
+        <hadoop.short.version>3.2</hadoop.short.version>
+        <commons-codec.version>1.11</commons-codec.version>
+        <metrics.version>3.2.4</metrics.version>
+        <bouncycastle.version>1.60</bouncycastle.version>
+        <junit4.version>4.11</junit4.version>
+      </properties>
+      <dependencyManagement>
+        <dependencies>
+          <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk15on</artifactId>
+            <version>${bouncycastle.version}</version>
+            <scope>test</scope>
+          </dependency>
+          <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
```

Comment:
Why do we need junit4? We have updated juni4 to junit5.

Reply:
Because hadoop-minicluster:3.2 depend on junit4. If junit4 is missing, some unit test will class not found exception.

Reply:
OK.